### PR TITLE
add DRM_fix_Component

### PR DESCRIPTION
**Pull request by Duo0405**
## Proposed changes
 
add DRM_fix_Component
 
## Types of changes
 
What types of changes does your code introduce to VIVEPORT?
_Put an `x` in the boxes that apply_
 
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
## Checklist
 
_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 

- [x] Feature or unit tests pass locally with my changes
- [ ] Code test coverage > 70% locally with my changes
- [x] I have run code inspection and shows no error
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+## For DRM wrapper Error, other use this VIVEPORT SDK DRM solution.
+
+* Download ViveportSDK Unreal. https://developer.viveport.com/documents/sdk/en/unrealengine.html
+* Download DRM_fix_Component.
+* Copy DRM_fix_Component\ViveportSDK folder to your Unreal project .\Plugins folder.
+* Change .\Plugins\ViveportSDK\Source\ViveportSDK\Classes\ViveportDRMComponent.h to your VIVEPORT ID and VIVEPORT KEY.
+* Remove .\Plugins\ViveportSDK\Binaries , .\Plugins\ViveportSDK\Intermediate , your Project's .\Binaries and your Project's .\Intermediate folder.
+* Start your .uproject. ureanl will start rebuild your project.
+* Create new Actor in your map and add Viveport DRM component . 
+* Run your project. If your viveport account doesn't have this app, Unral editor will end. This VIVEPORT SDK DRM is work.
```
Comment (Jake1006) [inline README.md]: Unreal

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# viveport_sdk_unreal_sample
\ No newline at end of file
+# VIVEPORT SDK Unreal Sample
+
+## For DRM wrapper Error, other use this VIVEPORT SDK DRM solution.
```
Comment (lyon-htc) [inline README.md]: If you encountered with VIVEPORT DRM wrapper compatibility issue in your unreal content, you can follow this sample to enable VIVEPORT DRM SDK.

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
...
+
+* Download ViveportSDK Unreal. https://developer.viveport.com/documents/sdk/en/unrealengine.html
+* Download DRM_fix_Component.
+* Copy DRM_fix_Component\ViveportSDK folder to your Unreal project .\Plugins folder.
+* Change .\Plugins\ViveportSDK\Source\ViveportSDK\Classes\ViveportDRMComponent.h to your VIVEPORT ID and VIVEPORT KEY.
+* Remove .\Plugins\ViveportSDK\Binaries , .\Plugins\ViveportSDK\Intermediate , your Project's .\Binaries and your Project's .\Intermediate folder.
+* Start your .uproject. ureanl will start rebuild your project.
```
Comment (lyon-htc) [inline README.md]: Unreal? Typo?

**Review by Jake1006 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lyon-htc) [inline README.md]: Run your project. If your VIVEPORT account doesn't own this app, Unreal editor will close. It means VIVEPORT DRM SDK is workable now.

File: DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    long long expiration_time,
+    int latest_version,
+    bool update_required
+    )
+{
+
+}
+
+void UViveportDRMComponent::MyLicenseChecker::OnFailure(int error_code, const FString& error_message) {
+    FString fstring = FString::Printf(TEXT("Verify failed!\n error_code=%d,\n error_message=%s"),
```
Comment (lyon-htc) [inline DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp]: Failed to verify the license.

File: DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        error_code,
+        *error_message);
+    FPlatformMisc::RequestExit(true);
```
Comment (lyon-htc) [inline DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp:46]: // Failed to verify the license, and the app will exit now. If you want to show some error messages or do something, you can change here.

File: README.md [hunk 6]
```diff
@@ -1 +1,12 @@
...
+* Create new Actor in your map and add Viveport DRM component .
```
Comment (lyon-htc) [inline README.md]: Create a new Actor in ....

File: DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+{
+    Super::EndPlay(EndPlayReason);
+
+    UViveportApi::Shutdown(&myShutdownCallback);
+}
+
+// Called every frame
+void UViveportDRMComponent::TickComponent(float DeltaTime, ELevelTick TickType, FActorComponentTickFunction* ThisTickFunction)
+{
+	Super::TickComponent(DeltaTime, TickType, ThisTickFunction);
```
Comment (lyon-htc) [inline DRM_fix_Component/ViveportSDK/Source/ViveportSDK/Private/ViveportDRMComponent.cpp]: Why do we need to tick it?

**Review by lyon-htc — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1 +1,12 @@
...
\ No newline at end of file
+# VIVEPORT SDK Unreal Sample
+
+## If you encountered with VIVEPORT DRM wrapper compatibility issue in your unreal content, you can follow this sample to enable VIVEPORT DRM SDK.
+
+* Download ViveportSDK Unreal. https://developer.viveport.com/documents/sdk/en/unrealengine.html
+* Download DRM_fix_Component.
+* Copy DRM_fix_Component\ViveportSDK folder to your Unreal project .\Plugins folder.
+* Change .\Plugins\ViveportSDK\Source\ViveportSDK\Classes\ViveportDRMComponent.h to your VIVEPORT ID and VIVEPORT KEY.
+* If .\Plugins\ViveportSDK\Binaries and .\Plugins\ViveportSDK\Intermediate, exist , place remove them. Then remove your Project's .\Binaries and your Project's .\Intermediate folder.
```
Comment (lyon-htc) [inline README.md]: If .\Plugins\ViveportSDK\Binaries and .\Plugins\ViveportSDK\Intermediate exist , **please** remove them. Then remove your Project's .\Binaries and your Project's .\Intermediate folder.

File: README.md (hunk 6 above)
Comment (lyon-htc) [reply README.md]: Add "a"

**Review by lyon-htc — CHANGES_REQUESTED** [review]

**Review by lyon-htc — APPROVED** [review]

**Pull request merged**